src/core/cardano/walletConnect/identityWalletConnect.ts

```diff
@@ -56,7 +54,7 @@ class IdentityWalletConnect extends CardanoPeerConnect {
         this.selectedAid
       );
       if (!identifier) {
-        throw new Error(IdentityWalletConnect.IDENTIFIER_ID_NOT_LOCATED);
+        throw new Error(`${Agent.MISSING_DATA_ON_KERIA}: ${this.selectedAid}`);
```

Comment:
Actually, we should remove the entire `if` block here. If it doesn't exist, it will have already been thrown in `getIdentifier`

Reply:
`getIdentifier` can return `undefined` in this case. If the `identifier` here is `undefined`, what should we do?
![image](https://github.com/user-attachments/assets/07ccc5a5-39f4-4ec7-8678-943e494da317)

Reply:
Ah, lets do something like this. We should never be calling this on a pending identifier so it's just a safety check.

```
static readonly IDENTIFIER_IS_PENDING = "Cannot fetch identifier details as the identifier is still pending";

if (metadata.isPending) {
  throw new Error(IdentifierService.IDENTIFIER_IS_PENDING);
}
```

I'm not sure why `signifyOpName` was in that if statement.

Can you add a test for this?

Reply:
In IdentifierDetails.tsx, can you update to this:
```
  const fetchDetails = useCallback(async () => {
    // @TODO - Error handling with try catch.
    const cardDetailsResult = await Agent.agent.identifiers.getIdentifier(
      params.id
    );
    setCardData(cardDetailsResult);
  }, [params.id]);
```

So the UI doesn't assume it's still `undefined` for not existing